docs/health-api-specs/contracts/stock.yml

```
@@ -1,818 +1,967 @@
-        $ref: '#/definitions/referenceIdType'
-      physicalCount:
-        $ref: '#/definitions/physicalCount'
-      calculatedCount:
-        $ref: '#/definitions/calculatedCount'
-      commentsOnReconciliation:
+          content:
+            '*/*':
+              schema:
+                $ref: >-
+                  https://raw.githubusercontent.com/egovernments/egov-services/master/docs/common/contracts/v1-1-1.yml#/definitions/ErrorRes
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)